src/components/loading/loading_content.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FunctionComponent, HTMLAttributes } from 'react';
+import classNames from 'classnames';
+import { CommonProps } from '../common';
+
+type LineRange = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10;
+
+export const EuiLoadingContent: FunctionComponent<
+  CommonProps &
+    HTMLAttributes<HTMLDivElement> & {
+      lines?: LineRange;
+    }
+> = ({ lines = 3, className, ...rest }) => {
+  const classes = classNames('euiLoadingContent', className);
+  const lineElements = [];
+
+  for (let i = 0; i < lines; i++) {
+    lineElements.push(
+      <div key={i} className="euiLoadingContent__singleLine">
+        <div className="euiLoadingContent__singleLine--background" />
```

Comment:
This is still a child so the class name should be 
```suggestion
        <div className="euiLoadingContent__singleLineBackground" />
```

Reply:
Haha. Will do. I'll understand the convention eventually. 😂